.github/workflows/publish_binary.yaml

```diff
@@ -54,3 +55,28 @@ jobs:
             ./.artifacts/pipectl_${{ env.PIPECD_VERSION }}_linux_arm64
             ./.artifacts/pipectl_${{ env.PIPECD_VERSION }}_darwin_amd64
             ./.artifacts/pipectl_${{ env.PIPECD_VERSION }}_darwin_arm64
+
+      # Building and publishing quickstart manifests.
+      - name: Install Helm
+        uses: Azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
+      - name: Build quickstart manifests
+        run: |
+          helm template pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version ${{ env.PIPECD_VERSION }} -n pipecd -f quickstart/control-plane-values.yaml > quickstart/manifests/control-plane.yaml
```

Comment:
`oci://ghcr.io/pipe-cd/chart/pipecd` is pushed in publish_image_chart.yaml. So I think we should place these codes in it.

https://github.com/pipe-cd/pipecd/blob/f017312df3e14251f9e65dfceb14e83c608b5805/.github/workflows/publish_image_chart.yaml#L134-L139

Reply:
@Warashi Got your point. I think I only want to trigger this workflow on release (the publish_image_chart trigger on every pull request which is merged to master), so I guess we should make a new workflow instead of adding to an existing one. WDYT? 🤔

Reply:
Changed my mind. I moved these steps to the publish_image_chart workflow as your suggestion, with some of the conditions checked in the steps' definition. Please recheck when you have time.